styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesPodEventTranslator.java

```diff
@@ -226,6 +226,12 @@ public TerminationLogMessage(
   private static Optional<Event> isInErrorState(WorkflowInstance workflowInstance, Pod pod) {
     final PodStatus status = pod.getStatus();
     final String phase = status.getPhase();
+    final String reason = status.getReason();
+
+    if (reason != null && "NodeLost".equals(reason)) {
```

Comment:
You write the equals in that backwards way to get rid of null checks 😄

Reply:
👍 :)